cirq/__init__.py

```diff
@@ -38,6 +38,7 @@
 from cirq import (
     # Core
     circuits,
+    qis,
```

Comment:
This should be lower. qis will be imported by more modules than this (e.g. sim would probably use it). Lower level goes lower.

Reply:
Lower means higher up in the file, right? I put it after linalg.